src/partials/dialog.settings.html

```diff
@@ -11,9 +11,47 @@ <h2 translate>settings.SETTINGS</h2>
         </md-toolbar>
         <md-dialog-content>
             <div class="md-dialog-content">
+                <section>
+                    <md-subheader class="md-accent"><span translate>settings.notifications.NOTIFICATIONS</span>
+                    </md-subheader>
+                    <md-list flex>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.notificationApiAvailable || ctrl.notificationPermission === false"
+                                    ng-model="ctrl.desktopNotifications"
+                                    ng-change="ctrl.setWantsNotifications(ctrl.desktopNotifications)"
+                                    aria-label="Show desktop notifications">
+                            </md-checkbox>
+                            <p translate>settings.notifications.SHOW_NOTIFICATIONS</p>
+                        </md-list-item>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.desktopNotifications"
```

Comment:
There's a little usability issue here: If the notifications are disabled (first option) and the preview is disabled too (second option), it's barely visible that the option is greyed out and cannot be checked.

Maybe we can hide the entire option if notifications are disabled?